SW-sdk/Services/Csd/CsdService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SW.Helpers;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace SW.Services.Csd
+{
+    public abstract class CsdService : Services
+    {
+        protected CsdService(string url, string user, string password, string proxy, int proxyPort) : base(url, user, password, proxy, proxyPort)
+        {
+        }
+        protected CsdService(string url, string token, string proxy, int proxyPort) : base(url, token, proxy, proxyPort)
+        {
+        }
+        internal abstract CargaCsdResponse CargaCsd(string cer, string key, string password, string certificateType, bool isActive);
+        internal virtual Dictionary<string, string> GetHeaders()
```

Comment:
Este objeto q estas creando en muchas clases no es necesario ya que todos deberian accederse como esta abajo

linea 32
 request.Headers.Add(HttpRequestHeader.Authorization.ToString(), "bearer " + this.Token);

Reply:
Done